packages/ui-components/package.json

```diff
@@ -15,10 +15,12 @@
     "test": "echo skipped"
   },
   "dependencies": {
-    "@polkadot/api-contract": "^0.90.0-beta.22",
-    "@polkadot/react-identicon": "^0.42.0-beta.15",
-    "@polkadot/ui-app": "^0.34.0-beta.107",
-    "@polkadot/ui-signer": "^0.34.0-beta.96",
+    "@polkadot/app-contracts": "^0.35.0-beta.6",
+    "@polkadot/app-staking": "^0.35.0-beta.6",
+    "@polkadot/react-components": "^0.35.0-beta.6",
+    "@polkadot/react-identicon": "^0.43.0-beta.0",
+    "@polkadot/react-params": "^0.35.0-beta.6",
+    "@polkadot/react-signer": "^0.35.0-beta.6",
```

Comment:
This is a bit unfortunate. We need `InputAddress`, which was in `ui-app`, but got renamed into `react-components`. But now I need to import all these other packages that are needed by `react-components`. Not sure what to do here...

Reply:
ah... I'm slightly more in favour of just making our own then, as we do with other components that are now in `react-components`, even though it'd be a pain in the neck, at least then we'll have more control going forward.